### PR TITLE
[fr] 3.0 sync

### PR DESCRIPTION
**Pull request by HavokInspiration**
Follows #4576 & #4587

### CODE REVIEW

File: fr/controllers/middleware.rst [hunk 1]
```diff
@@ -282,6 +282,10 @@ application.
 Once those two steps are complete, you are ready to start re-implementing any
 application/plugin dispatch filters as HTTP middleware.
 
+Si vous exécutez des tests, vous aurez aussi besoin de mettre à jour votre
+fichier ``tests/bootstrap.php`` en récupérant le contenu du fichier du
```
Comment (antograssiot) [inline fr/controllers/middleware.rst]: Filenames are supposed to be in bold, not between backtik.

File: fr/core-libraries/global-constants-and-functions.rst [hunk 2]
```diff
@@ -143,6 +143,14 @@ la traduction de contenu.
     Communément utilisé comme ceci
     ``list($namespace, $className) = namespaceSplit('Cake\Core\App');``
 
+.. php:function:: dd(mixed $var, boolean $showHtml = null)
+
+    Cette méthode fonctionne comme ``debug()`` sauf qu'elle arrêtera
+    l'exécution du script.
+    Si la variable "core" ``$debug`` vaut ``true``, ``$var`` sera affichée.
+    Si ``$showHTML`` vaut ``true`` ou laisser à ``null``, les données seront
```
Comment (antograssiot) [inline fr/core-libraries/global-constants-and-functions.rst]: laissée

File: fr/core-libraries/global-constants-and-functions.rst [hunk 3]
```diff
@@ -143,6 +143,14 @@ la traduction de contenu.
...
+    rendus dans un affichage *user-friendly*. Plus de détails : :doc:`/development/debugging`
```
Comment (antograssiot) [inline fr/core-libraries/global-constants-and-functions.rst]: rendues  ?

File: fr/controllers/middleware.rst (hunk 1 above)
Comment (HavokInspiration) [inline fr/controllers/middleware.rst]: I'll fix the original version as well.

**Comment by HavokInspiration** [comment]
@antograssiot Fixed based on your review

**Comment by antograssiot** [comment]
Nice @HavokInspiration

**Pull request merged**